### PR TITLE
[Fix] Media card url fix

### PR DESCRIPTION
**Pull request by AlexanderKanakis**
Changed custom payload object property "imageCardUrl" to "mediaCardUrl" to match the changes in RC's customFields handling.

### CODE REVIEW

**Comment by ear-dev** [comment]
@AlexanderKanakis can you add some description please as to what is in this PR?
Thanks!

**Comment by AlexanderKanakis** [comment]
@ear-dev Done!

File: handler/PostMessageSentHandler.ts [hunk 1]
```diff
@@ -86,8 +86,9 @@ export class PostMessageSentHandler {
         const { visitor: { token: visitorToken } } = room as ILivechatRoom;
 
         try {
-            await botTypingListener(rid, this.modify.getNotifier().typing({ id: rid, username: DialogflowBotUsername }));
-            response = (await Dialogflow.sendRequest(this.http, this.read, this.modify, this.persistence, rid, text, DialogflowRequestType.MESSAGE));
+            await botTypingListener(this.modify, rid, DialogflowBotUsername);
+            response = (await Dialogflow.sendRequest(this.http, this.read, this.modify, rid, text, DialogflowRequestType.MESSAGE));
```
Comment (Shailesh351) [inline handler/PostMessageSentHandler.ts:90]: 4th argument `this.persistence` is missing here. Isn't it giving you an error?

Comment (AlexanderKanakis) [reply handler/PostMessageSentHandler.ts:90]: @Shailesh351 you can ignore this PR, I will make the changes in a new one. Should I pull from cx_development or master?

Comment (Shailesh351) [reply handler/PostMessageSentHandler.ts:90]: One PR for merging master in cx_development. And one for this mediaCardURL?

Comment (AlexanderKanakis) [reply handler/PostMessageSentHandler.ts:90]: I want to do a small PR to fix Steve's issue from yesterday. I will just update this PR to be only about changes for mediaCardURL.

**Pull request closed without merging**